src/main/java/de/tum/cit/aet/artemis/nebula/dto/FaqConsistencyResponse.java

```diff
@@ -0,0 +1,9 @@
+package de.tum.cit.aet.artemis.nebula.dto;
+
+import java.util.List;
+
+import jakarta.annotation.Nullable;
+import jakarta.validation.constraints.NotNull;
+
+public record FaqConsistencyResponse(@NotNull boolean consistent, @Nullable List<String> inconsistencies, @Nullable String improvement, @Nullable List<Long> faqIds) {
```

Comment:
Why don't you call this DTO a DTO?
```suggestion
public record FaqConsistencyResponseDTO(@NotNull boolean consistent, @Nullable List<String> inconsistencies, @Nullable String improvement, @Nullable List<Long> faqIds) {
```